components/proposal-progress-bar.tsx

```diff
@@ -62,7 +63,7 @@ export default function ProposalProgressBar({
   function loadDisputeState() {
     setIssueState(getStateText());
     setIssueColor(getStateColor());
-    setPercentage(+toPercent(issueDisputeAmount, state.currentUser?.balance?.staked?.toNumber()));
+    setPercentage(toPercent(issueDisputeAmount, state.currentUser?.balance?.staked?.toNumber()));
```

Comment:
staked amount is no longer used, the network TVL is `state.Service.network.amounts.totalNetworkToken`
https://github.com/bepronetwork/web-network/blob/dev/interfaces/application-state.ts#LL48C11-L48C11